src/collector.js

```diff
@@ -22,6 +24,14 @@ export default class Collector {
   }
 
   _addMessage(type, opts, _Message=Message) {
+    // Filter the messages reported by file when the Linter has been configured
```

Comment:
@EnTeQuAk how about this inline comment? does it help to grasp the role of this check without too much thinking?